packages/uui-resize-observer/lib/uui-resize-observer.element.ts

```diff
@@ -0,0 +1,91 @@
+import { defineElement } from '@umbraco-ui/uui-base/lib/registration';
+import { property } from 'lit/decorators.js';
+import { css, html, LitElement } from 'lit';
+
+import { UUIResizeEvent } from './UUIResizeEvent';
+
+/**
+ * @element uui-resize-observer
+ */
+@defineElement('uui-resize-observer')
+export class UUIResizeObserverElement extends LitElement {
+      static styles = [
+    css`
+      :host {
+        display: contents;
+      }
+    `,
+  ];
+
+  private _resizeObserver!: ResizeObserver;
+  private _observedElements: HTMLElement[] = [];
+
+  /** Disables the observer. */
+  @property({ type: Boolean, reflect: true }) disabled = false;
+
+  connectedCallback() {
+    super.connectedCallback();
+    this._resizeObserver = new ResizeObserver((entries: ResizeObserverEntry[]) => {
+      this.dispatchEvent(new UUIResizeEvent(UUIResizeEvent.CHANGE));
```

Comment:
@iOvergaard would `CustomEvent` be better instead of a `UUIResizeEvent`?

Reply:
We tend to use UUI-events inside the library. They ultimately extend Event. I think it's perfectly fine what you are doing here.